multicore_runtime/wsgi_test.py

```diff
@@ -77,55 +90,55 @@
 logging.basicConfig(level=logging.CRITICAL)
 
 
-@Request.application
+@wrappers.Request.application
 def hello_world(request):  # pylint: disable=unused-argument
-  return Response(HELLO_STRING)
+  return wrappers.Response(HELLO_STRING)
 
 
-@Request.application
+@wrappers.Request.application
 def dump_os_environ(request):  # pylint: disable=unused-argument
-  return Response(json.dumps(dict(os.environ)))
+  return wrappers.Response(json.dumps(dict(os.environ)))
 
 
-@Request.application
+@wrappers.Request.application
 def add_to_os_environ(request):  # pylint: disable=unused-argument
   os.environ['ENVIRONMENT_MODIFIED'] = 'TRUE'
-  return Response(json.dumps(dict(os.environ)))
+  return wrappers.Response(json.dumps(dict(os.environ)))
 
 
-@Request.application
+@wrappers.Request.application
 def wait_on_global_event(request):  # pylint: disable=unused-argument
   concurrent_request_is_started.set()
   concurrent_request_should_proceed.wait()
-  return Response(json.dumps(dict(os.environ)))
+  return wrappers.Response(json.dumps(dict(os.environ)))
 
 
-@Request.application
+@wrappers.Request.application
 def sort_os_environ_keys(request):  # pylint: disable=unused-argument
   # See test_env_sort method for explanation.
-  resp = ""
-  for name in sorted(os.environ):
-    resp += '%s=%s\n' % (name, os.environ[name])
-  return Response(resp)
+  return wrappers.Response(''.join(
+      '%s=%s\n' % (k, v) for k, v in sorted(os.environ.iteritems())
+      ))
 
 
 class MetaAppTestCase(unittest.TestCase):
+  # pylint: disable=g-import-not-at-top
```

Comment:
I believe imports are only expected in `setUp` so you could perhaps move this there, or perhaps even only on the two import lines (`import vmconfig` and `import wsgi`).